data/json/item_groups.json

```diff
@@ -1748,6 +1748,26 @@
       [ "veggy_canned", 1 ]
     ]
   },
+  {
+    "type": "item_group",
+    "id": "cannedmeat",
```

Comment:
This new item group doesn't seem to be used anywhere.

Reply:
Removed and slated for another PR